gpytorch/__init__.py

```diff
@@ -119,7 +119,41 @@ def mvn_kl_divergence(mean_1, chol_covar_1, mean_2, covar_2):
 
     For more information, see the MVNKLDivergence function documentation.
     """
-    return MVNKLDivergence()(mean_1, chol_covar_1, mean_2, covar_2)
+    if isinstance(covar_2, LazyVariable):
+        return covar_2.mvn_kl_divergence(mean_1, chol_covar_1, mean_2)
```

Comment:
I understand why the arguments are in this order, but it's weird that `covar_2` has the `mvn_kl_divergence` function, since normally the first argument is the one that makes the function call.

If there's not a better way, we can keep it this way, but this is a bit of an antipattern.